control/Controller.php

```diff
@@ -83,30 +145,59 @@ public function Link() {
 	}
 
 	/**
+	 * {@inheritdoc}
+	 *
+	 * Also set the URLParams
```

Comment:
If the intention is to add detail to the inherited phpdoc then it is used correctly.